README.md

```diff
@@ -1,136 +1,30 @@
-bosh-softlayer-cpi [![Build Status](https://travis-ci.org/cloudfoundry/bosh-softlayer-cpi.svg?branch=master)](https://travis-ci.org/cloudfoundry/bosh-softlayer-cpi#)
-==================
+# BOSH Softlayer CPI Release
 
-An external [BOSH](http://github.com/cloudfoundry/bosh) CPI (cloud provider interface) for the [SoftLayer](http://www.softlayer.com) cloud and its [API](http://sldn.softlayer.com/article/SoftLayer-API-Overview)
+* Documentation: [bosh.io/docs](https://bosh.io/docs)
+* BOSH Slack channel: [#bosh](https://cloudfoundry.slack.com/archives/bosh)
+* BOSH SoftLayer CPI Slack channel: [#bosh-softlayer-cpi](https://cloudfoundry.slack.com/archives/bosh-softlayer-cpi)
+* Mailing list: [cf-bosh](https://lists.cloudfoundry.org/pipermail/cf-bosh)
+* CI: <http://bosh-softlayer-cpi.ci.bluemix.net/pipelines/bosh-softlayer-cpi-release-v2>
+* Roadmap: [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1344876)
 
-## Getting Started (*)
-------------------
+## Releases
 
-TBD
+This is a BOSH release for the Softlayer CPI.
 
-### Overview Presentations (*)
---------------------------
+The latest version for the SoftlLayer CPI release is here. Also, it is already available on [bosh.io](http://bosh.io).
 
-1. CF Summit 05/12/2015: [bosh-init](https://github.com/maximilien/presentations/blob/master/2015/cf-summit-bosh/releases/bosh-init-v1.0.0.pdf) - which also includes some details around BOSH external CPIs (motivation and goals)
+To use this CPI you will need to use the SoftLayer light stemcell. it's also available on [bosh.io](http://bosh.io).
 
-### Cloning and Building
-------------------------
+## Bootstrap on SoftLayer
 
-Clone this repo and build it. Using the following commands on a Linux or Mac OS X system:
+You can use bosh-init from community to bootstrap an environment.
```

Comment:
Yes, I agree. Since we are combining two projects into one, I think it's better to rewrite and merge the README.md based on this new repository and other updates(like CLI2).
I opened a chore to track this.[#137845019](https://www.pivotaltracker.com/story/show/137845019)